platformio.ini

```diff
@@ -14,7 +14,7 @@
 ;env_default = esp32dev
 ;env_default = dev_ESP8266_4096
 ;env_default = normal_ESP8266_1024
-;env_default = normal_ESP8266_4096
+env_default = normal_ESP8266_4096
```

Comment:
Not sure what it was, but a (long) time ago, we put those default settings in comments.
I cannot remember the reason why.

Reply:
I will revert that change